docker-compose.yaml

```diff
@@ -23,12 +23,20 @@ services:
     container_name: batch-cloud
     healthcheck:
       test: "exit 0"
+    volumes:
+      - ./utils/dbz_event_to_fhir_config.json:/usr/src/Main/dbz_event_to_fhir_config.json
+      # data is the directory which you want to persist the generated parquet files
+      - ./data:/tmp
     environment:
      - OPENMRS_URL=http://localhost:8099/openmrs
-     - SINK_PATH=http://localhost:8098/fhir
```

Comment:
Question: What is the rational for specifying some env. vars here (like `BATCH_SIZE`) but not others (like `SINK_PATH`)? Would that make sense to simply pass all relevant env. vars and also just use the defaults in the Dockerfile like [here](https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers)?